ecommerce_extensions/tenant/receivers.py

```diff
@@ -15,16 +18,25 @@
 
 
 def update_tenant_settings(sender, environ, **kwargs):  # pylint: disable=unused-argument
-    """
-    Update the general django settings with the found values in TenantOptions.
-    """
+    """Perform update and reset actions."""
     http_host = environ.get("HTTP_HOST")
 
     if not http_host:
         LOG.warning("Could not find the host information.")
         return
 
-    options = TenantAwareSettings.get_tenant_options(http_host)
+    domain = http_host.split(':')[0]
+
```

Comment:
Why not using just double quotes instead of single and double?

Reply:
done